src/server/tests.rs

```diff
@@ -1392,11 +1396,15 @@ fn copy_from_wayland<T: SelectionTest>() {
             s.spawn(|| {
                 let pollfd = unsafe { BorrowedFd::borrow_raw(f.testwl.poll_fd().as_raw_fd()) };
                 let mut pollfd = [PollFd::from_borrowed_fd(pollfd, PollFlags::IN)];
-                if poll(&mut pollfd, 100).unwrap() == 0 {
+                let timeout = Some(&Timespec {
+                    tv_sec: 0,
+                    tv_nsec: 100_000_000,
```

Comment:
```suggestion
                    tv_nsec: std::time::Duration::from_millis(100).as_nanos()
```
This could also go in a `const`.